apertium-crh.crh.lexc

```diff
@@ -18,107 +23,119 @@ Multichar_Symbols
 %<adv%>           ! Adverb                      !
 %<v%>             ! Verb                        !
 %<cnjcoo%>        ! Coordinating conjunction    !
-%<cnjsub%>        ! "subordinating conjunction" (ki)
-%<cnjadv%>        ! 
-%<qst%>           ! Question word
-%<sym%>
+%<cnjsub%>        ! Subordinating conjunction (ki)
+%<cnjadv%>        ! Adverbial conjunction
+%<qst%>           ! Modal question particle     !# м{I}
+                  
+%<sym%>           ! Symbol
 %<cop%>           ! Copula
-%<postadv%>       ! 
-%<det%>           ! 
-%<num%>           ! 
+%<postadv%>       ! Post of Adverb
+%<det%>           ! Determiner
+%<num%>           ! Numeral
 %<vaux%>          ! Auxiliary verb
 %<abbr%>          ! Abbreviation
-%<emph%> 
-%<ij%>
-%<nom%>           ! Nominative                  !
-%<gen%>           !
-%<dat%>           !
-%<acc%>           !
-%<loc%>           ! Locative                    ! 
-%<abl%>           ! Ablative                    !
-%<ins%>           !
-%<sim%>
-
-%<iv%>            !
-%<tv%>            !
-
-%<nec%>
-%<imp%>
-%<aor%>
-%<gno%>
-%<fut%>
+%<emph%>          ! Emphasizing modal particle  !# -ч{I}, -с{A}н{A}, әле
+%<ij%>            ! Interjection
+%<nom%>           ! Nominative
+%<gen%>           ! Genitive
+%<dat%>           ! Dative
+%<acc%>           ! Accusative
+%<loc%>           ! Locative
+%<abl%>           ! Ablative
+%<ins%>           ! Instrumental=comitative
+%<sim%>           ! Similative
+
+! Transitivity
+%<iv%>            ! Transitive
+%<tv%>            ! Intransitive
+
+%<nec%>           ! Necessitive
+%<imp%>           ! Imperative
+%<aor%>           ! Aorist
+%<gno%>           ! Gnomic tense
+%<fut%>           ! -{I}р
 %<fut2%>          ! -EcAG
-%<past%>
-%<ifi%>
```

Comment:
These two tags have been removed and should be added back.

Reply:
Ok added them back and commented on them.